### PR TITLE
removing plugin code

### PR DESCRIPTION
**Pull request by ScriptedAlchemy**
Now that we depend entirely on webpack core. Theres no more need for the plugins workspaces or their code. Only the examples

### CODE REVIEW

File: examples/website1/package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
     "html-webpack-plugin": "^4.0.0-beta.11",
     "serve": "^11.3.0",
     "webpack": "git://github.com/ScriptedAlchemy/webpack.git#fixing-overrides",
-    "webpack-cli": "^3.3.11",
-    "webpack-external-import": "0.0.0-development"
+    "webpack-cli": "^3.3.11"
```
Comment (maraisr) [inline examples/website1/package.json]: Should this not be using the beta also?

```suggestion
    "webpack-cli": "^4.0.0-beta.3"
```

File: examples/website2/package.json [hunk 2]
```diff
@@ -18,7 +18,6 @@
     "serve": "^11.3.0",
     "html-webpack-plugin": "^4.0.0-beta.11",
     "webpack": "git://github.com/ScriptedAlchemy/webpack.git#fixing-overrides",
-    "webpack-cli": "^3.3.11",
-    "webpack-external-import": "0.0.0-development"
+    "webpack-cli": "^3.3.11"
```
Comment (maraisr) [inline examples/website2/package.json]: same here
```suggestion
    "webpack-cli": "^4.0.0-beta.3"
```

File: examples/small-spa/package.json [hunk 3]
```diff
@@ -9,7 +9,6 @@
     "css-loader": "^3.4.2",
     "style-loader": "^1.1.3",
     "webpack": "git://github.com/ScriptedAlchemy/webpack.git#fixing-overrides",
```
Comment (maraisr) [inline examples/small-spa/package.json]: Seeing as its in dev-1 now, we could probably just point this repo at the official one?

File: examples/website1/package.json (hunk 1 above)
Comment (ScriptedAlchemy) [reply examples/website1/package.json]: tobias downgraded it :P so apparently not haha

File: examples/website2/package.json (hunk 2 above)
Comment (ScriptedAlchemy) [reply examples/website2/package.json]: same as above

File: examples/small-spa/package.json (hunk 3 above)
Comment (ScriptedAlchemy) [reply examples/small-spa/package.json]: yeah agreed

**Pull request merged**